### ISSUE TITLE
[6주차] Time Series, NN Advanced 과제 제출 (~2/27) 

### ISSUE THREAD

**Issue by ymnseol** [tobigs-datamarket/tobigs-21st#86]
# Week6. Time Series, Neural Network Advanced

## Part 1. Time Series

`Week6_Time Series Analysis_Assignment.ipynb`에서 주가 데이터를 가지고,

- [ ] Box-Jenkins 방법론 과정을 따라 시계열분석 진행하고 결과 예측하기

### 이론

다음의 내용을 주제로 하는 보고서 자유 형식으로 작성하기

- [ ] 다양한 Learning Rate Scheduler 중 적어도 한 가지 소개하기 (A4 반 페이지 내외)
- [ ] 강의에서 소개되지 않은 Training Error와 Generalization Error 사이 간극을 줄이는 방안 소개하기 (A4 반 페이지 내외)

### 실습
`NN_Adv_Assignment.ipynb`에,

- [ ] 적절한 Optimization, Regularization, Initialization, Normalization 전략을 각각 최소 1개씩 수행하기
- [ ] 15 에포크 이내에 제공된 데이터셋에서 Validation Accuracy 80%를 넘기기
- [ ] Ipynb 파일 마지막에 본인이 수행한 전략과 본인만의 분석이 담긴 나름의 ‘결론부‘ 작성하기

기본적으로 실습과제 성능 순으로 우수과제를 선별하며, 이론 과제 및 실습 결론부가 인상적일 경우 가산점을 부여합니다.

⚠️ `Train.csv`, `Test.csv`는 카카오톡으로 공지된 강의자료 링크에서 다운로드하여 사용해주시기 바랍니다.

## 제출 방법
1. 본인의 이름으로 branch를 생성합니다.
2. `main`을 본인의 branch로 pull합니다.
3. **본인의 branch에서** 과제 디렉토리에 본인의 이름으로 디렉토리를 생성합니다.
4. 과제를 수행합니다. 디렉토리 구조:
   ```
   tobigs-21st
   └── Week1
       ├── EDA
       │   └── 김투빅
       │       └── example.md # 과제 파일
       └── Framework
           └── 김투빅
               └── example.md # 과제 파일
   ```
5. 변경된 본인의 branch를 commit, push합니다.
6. `이름(기수) - n주차 과제`라는 타이틀로 `Pull Request`를 등록합니다. 예) 김투빅(21기) - 1주차 과제

## 제출 기한
⏰ ~ 2/27(화) 23:59
